app/models/manageiq/providers/cloud_manager/provision_workflow.rb

```diff
@@ -92,11 +92,11 @@ def availability_zone_to_cloud_network(src)
     if src[:availability_zone]
       load_ar_obj(src[:availability_zone]).cloud_subnets.each_with_object({}) do |cs, hash|
         cn = cs.cloud_network
-        hash[cn.id] = cn.name
+        hash[cn.id] = cn.cidr.blank? ? cn.name : "#{cn.name} (#{cn.cidr})"
       end
     else
       load_ar_obj(src[:ems]).all_cloud_networks.each_with_object({}) do |cn, hash|
-        hash[cn.id] = cn.name
+        hash[cn.id] = cn.cidr.blank? ? cn.name : "#{cn.name} (#{cn.cidr})"
```

Comment:
@lfu Would suggest DRYing up this naming logic since it gets repeated here as well as in some external repos.

```ruby
def cloud_network_display_name(cn)
  cn.cidr.blank? ? cn.name : "#{cn.name} (#{cn.cidr})"
end
```